app/helpers/operatingsystems_helper.rb

```diff
@@ -13,49 +13,49 @@ def icon(record, opts = {})
     return "" if record.blank? || record.name.blank?
     size = opts[:size] ||= '16x16'
     name = case record.name
```

Comment:
while we're here, you could do 
```suggestion
    name = case record.name.downcase
```
and drop the `/i` matching everywhere (and make all the matches downcase).

Reply:
Yeah, sure :) why not :+1: